src/System.Runtime.Extensions/src/System/IO/Path.cs

```diff
@@ -49,13 +49,13 @@ public static class Path
 #endif // !PLATFORM_UNIX
 
         // Platform specific alternate directory separator character.  
-        // This is backslash ('\') on Unix, and slash ('/') on Windows 
-        // and MacOS.
+        // This is slash ('/') on Windows and MacOS.
+        // There is no alternate separator on Unix, so the normal separator is used.
         // 
 #if !PLATFORM_UNIX        
         public static readonly char AltDirectorySeparatorChar = '/';
 #else
-        public static readonly char AltDirectorySeparatorChar = '\\';
+        public static readonly char AltDirectorySeparatorChar = '/';
 #endif // !PLATFORM_UNIX
```

Comment:
This now looks odd.  If we're !UNIX its '/' else its '/'?  Can't we just remove this and clean up the file?